tests/cpp/helpers.h

```diff
@@ -154,7 +154,8 @@ class SimpleRealUniformDistribution {
 std::shared_ptr<xgboost::DMatrix> *CreateDMatrix(int rows, int columns,
                                                  float sparsity, int seed = 0);
 
-std::unique_ptr<DMatrix> CreateSparsePageDMatrix();
+std::unique_ptr<DMatrix> CreateSparsePageDMatrix(const size_t n_entries=12,
```

Comment:
Is it possible to drop the default values?

Also, no need for `const size_t` in parameters, just `size_t` is fine.

Reply:
Done.